### ISSUE TITLE
Please remove "lynch" from word list

### ISSUE THREAD

**Issue by petermalcolm** [kmc059000/octordle#17]
I love octordle and other family members play with me sometimes -- and my mom and school-age son have gotten hooked too. 
But some words kill the fun, and "lynch" is such a word.

**Comment by kmc059000** [comment]
tldr: I agree, but it is going to take some time to do that due to technical challenges. I ask you and anybody else concerned about this topic to be patient.

Long answer:
I hear you, there are a few words that can be answers that I wish weren't. I didn't come up with the original list.

The challenge is the way the randomization works for the daily game isn't easily changeable in a way that keeps it consistent across deploys of the app. As it stands now, if I add or remove a word, the words end up completely different for the same daily game before and after the change.

A big problem that exists is that people open the game in a tab, and then literally never refresh it for weeks at a time. In one instance, this led to there being 2 different sets of words for a daily game which led to confusion from people.

So the goal here is to be able to remove only the censored words for people using future versions, while leaving the rest of the words exactly the same in a daily game. This will reduce the differences in words between versions and lead to less chaos across the octordleverse. 

On top of that, there are 2 versions of octordle and v2 is actively in the works which makes it hard to make this change to keep the daily words in sync across v1 and v2.

Fortunately the chances of these certain words being selected are low, roughly .2% chance for the handful of words I know about.

**Comment by kmc059000** [comment]
I added censored words a while back and this word was included among a handful of others. Thanks for the report.
